examples/qvm/multishot.c

```diff
@@ -82,7 +82,8 @@ void multishot_with_implicit_ro_indices() {
 
   qvm_multishot_result qvm_res;
   int num_trials = 10;
-  if (qvm_multishot(program, addresses, num_trials, &qvm_res) !=
+  double gate_noise[] = {0.0, 0.0, 0.0};
+  if (qvm_multishot(program, addresses, num_trials, &qvm_res, NULL, NULL) !=
```

Comment:
Curious why the arg ordering is different than L39 above?

And neither appears to match the signature defined in the Lisp change in this PR but I assume there's an FFI reason for that

Reply:
That particular example was incorrect (the result always comes in the last position). In the Lisp code, the result is not always included as a parameter in the function signature, in which case there is an implicit argument which takes its value from the result of the corresponding Lisp function.